packages/timeline-state-resolver/src/devices/vizMSE.ts

```diff
@@ -1324,11 +1326,11 @@ class VizMSEManager extends EventEmitter {
 		return `sofieInt_${layer.templateName}_${getHash((layer.templateData ?? []).join(','))}`
 	}
 
-	static getPlayoutItemContent(playoutItem: VIZMSEPlayoutItemContent): VizMSEPlayoutItemContentInstance {
+	static getPlayoutItemContent(this: void, playoutItem: VIZMSEPlayoutItemContent): VizMSEPlayoutItemContentInstance {
```

Comment:
What is the purporse of including "this" as a parameter? It is not used in the implementation?